### ISSUE TITLE
TypeError when sending query using python3

### ISSUE THREAD

**Issue by PidgeyL** [osquery/osquery-python#16]
running this code:

```
import osquery
CLIENT = osquery.ExtensionClient()
CLIENT.open()
RESULTS = CLIENT.extension_client().query(sys.argv[1])
```

results in the following error:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.4/dist-packages/osquery-1.5.3-py3.4.egg/osquery/extensions/ExtensionManager.py", line 183, in query
  File "/usr/local/lib/python3.4/dist-packages/osquery-1.5.3-py3.4.egg/osquery/extensions/ExtensionManager.py", line 187, in send_query
  File "/usr/local/lib/python3.4/dist-packages/thrift/protocol/TBinaryProtocol.py", line 46, in writeMessageBegin
    self.writeI32(TBinaryProtocol.VERSION_1 | type)
  File "/usr/local/lib/python3.4/dist-packages/thrift/protocol/TBinaryProtocol.py", line 111, in writeI32
    self.trans.write(buff)
  File "/usr/local/lib/python3.4/dist-packages/thrift/transport/TTransport.py", line 168, in write
    raise e
  File "/usr/local/lib/python3.4/dist-packages/thrift/transport/TTransport.py", line 164, in write
    self.__wbuf.write(buf)
TypeError: string argument expected, got 'bytes' 
```

when ran using python3.

Should I run this code differently, when using python3, or is this a bug?

Thanks in advance

**Comment by theopolis** [comment]
Hey @PidgeyL, I haven't tested, but by looking at the stack trace real fast, maybe this is a false expectation by the thrift code. Perhaps: http://stackoverflow.com/questions/31869321/thrift-python-3-4-typeerror-string-argument-expected-got-bytes or http://stackoverflow.com/questions/32229690/apache-thrift-python-3-support

Looks like there's a Thrift 0.9.3 released 20 days ago: https://github.com/apache/thrift/releases/tag/0.9.3 maybe give that a go? I can also try/debug later tonight when I'm home.

**Comment by PidgeyL** [comment]
Hello @theopolis, I already had version 0.9.3 installed. I tried to downgrade, and that didn't work either. For me, it's night time though. Thanks in advance, if you need me to do some tests, I'll take a look tomorrow

**Comment by theopolis** [comment]
This should be fixed in version 3.0.0.
